vendor/mcci/catena4430-profile.yaml

```diff
@@ -0,0 +1,53 @@
+# Vendor profile ID, can be freely issued by the vendor
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
+# This is a decimal number, but it's limited to 0..65535
+vendorProfileID: 1
+
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: 1.0.3
+# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
+#   1.0:   TS001-1.0
+#   1.0.1: TS001-1.0.1
+#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
+#   1.0.3: RP001-1.0.3-RevA
+#   1.0.4: RP002-1.0.0 or RP002-1.0.1
+#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
+regionalParametersVersion: RP001-1.0.3-RevA
+
+# Whether the end device supports join (OTAA) or not (ABP)
+supportsJoin: true
+# If your device is an ABP device (supportsJoin is false), uncomment the following fields:
+# RX1 delay
+#rx1Delay: 5
+# RX1 data rate offset
+#rx1DataRateOffset: 0
+# RX2 data rate index
+#rx2DataRateIndex: 0
+# RX2 frequency (MHz)
+#rx2Frequency: 868.525
+# Factory preset frequencies (MHz)
+#factoryPresetFrequencies: [868.1, 868.3, 868.5, 867.1, 867.3, 867.5, 867.7, 867.9]
+
+# Maximum EIRP
+maxEIRP: 16
```

Comment:
Are you using this for all your regions? Because all regions refer to this file.

Reply:
Well... we're using the same Murata module everywhere. The max power I've measured on our modules with the RWC5020 is pretty low, 16 to 18 dBm. And then EIRP depends on the (customer) antenna. Is this supposed to be a regulatory thing or a report of fact about our software/hardware combination. (I had a massive amount of data to merge, so rather than I went on, rather than asking.)

In fact, my team did a review this morning and found a few broken links, etc. So before merging, we might want to fix those. Based on what `maxEIRP` is supposed to represent, we can fix that at the same time.  (We'll have the same profile ID on each of these thing independent of region, so this would just be a matter of cloning files and adjusting.)

Reply:
converting to draft since we want to touch up a few links.

Reply:
Thanks for your quick response.

> Is this supposed to be a regulatory thing or a report of fact about our software/hardware combination

It's the latter. I was mostly asking if you were sure; I'm fine merging like this if you fixed the links.

When you mark the PR as ready for review, I'll have a look again.